### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by SlivnoeDulo** [community/community#48309] [Programming Help]
### Select Topic Area

Bug

### Body

Hello! I’m new to the unreal engine and When I try to create a new c++ project I have a bug: "The following modules are missing or built with a different engine version" and if I press "yes" I see an error: "could not be compiled. Try rebuilding from source manually" if I press "no" nothing happens. And I don’t know which plugin is required to solve this problem.

**Comment by queenofcorgis** [comment]
Hi @SlivnoeDulo ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the  [Unreal Engine Forum](https://forums.unrealengine.com/categories?tag=unreal-engine). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
